addins/Cake.Boots.yml

```diff
@@ -0,0 +1,13 @@
+Name: Cake.Boots
+NuGet: Cake.Boots
+Assemblies:
+- "/**/Cake.Boots.dll"
+Repository: https://github.com/jonathanpeppers/boots
+Author: jonathanpeppers
+Description: "boots is a dotnet global tool for "bootstrapping" vsix & pkg files."
```

Comment:
Guess this isn't escaped correctly?
```diff
- ((Line: 7, Col: 50, Idx: 202) - (Line: 7, Col: 83, Idx: 235): While parsing a block mapping, did not find expected key.)
```

Reply:
What is the proper escaping? Replacing the double-quote character with `\"`? Any other characters that need to be escaped? I have been searching Wyam's web site but I can't seem to find documentation about characters that need escaping.

Reply:
We're using YamlDotNet to parse ( https://github.com/cake-build/website/blob/726566df8ee5b4770850881db43669b3561d78ae/build.cake#L5 )
Which serializer are you using to generate yaml?

Reply:
I am generating the YAML by hand, without a serializer.

I just ran the following test with YamlDotNet's serializer to see if it would solve the problem:
```
using (var writer = new StringWriter())
{
    var myObject = new
    {
        Description = "This is a test containing \"double quotes\"."
    };
    var serializer = new SerializerBuilder().Build();
    serializer.Serialize(writer, myObject);
    Console.Out.Write(writer.ToString());
}
```
and, unfortunately, the result is:
`Description: This is a test containing "double quotes".`

Are there settings I should configure to ensure double quotes are escaped?

Reply:
@devlead are you aware of anything that can help here?

Reply:
Valid yaml is
```yml
Name: Cake.Boots
NuGet: Cake.Boots
Assemblies:
- "/**/Cake.Boots.dll"
Repository: https://github.com/jonathanpeppers/boots
Author: jonathanpeppers
Description: "boots is a dotnet global tool for \"bootstrapping\" vsix & pkg files."
Categories:
- Bootstrap
- CI
- Xamarin
- Azure
- DevOps
```
or
```yml
Name: Cake.Boots
NuGet: Cake.Boots
Assemblies:
- "/**/Cake.Boots.dll"
Repository: https://github.com/jonathanpeppers/boots
Author: jonathanpeppers
Description: boots is a dotnet global tool for "bootstrapping" vsix & pkg files.
Categories:
- Bootstrap
- CI
- Xamarin
- Azure
- DevOps
```

So what YamlDotNet produces is correct, and it parses both fine.

Current `Description: "boots is a dotnet global tool for "bootstrapping" vsix & pkg files."` is not valid.

Reply:
oh I see! I didn't realize that a string not enclosed in quotes was valid. I'll make the correction in my code.

On a side note: while playing with the YamDotNet serialize I noticed that it uses the windows line terminator(`\r\n`). Do you know if there is a way to configure it to use a different line terminator? (files in the Cake website repo use `\n`).